misc/python/materialize/data_ingest/executor.py

```diff
@@ -133,6 +133,7 @@ def __init__(
             registry, json.dumps(key_schema), lambda d, ctx: d
         )
 
+        # TODO: Is this correct at all? Should this be self.topic? Check with def-
```

Comment:
I think I commented on this elswhere -- you probably need to put this inside the `for topic` loop.